### PR TITLE
Create leonardotmnt.yml

### PR DESCRIPTION
**Pull request by superiorxman**

### CODE REVIEW

**Comment by superiorxman** [comment]
https://www.youtube.com/watch?v=oYa6XWIn4Vo
0:50-1:04
katanas
jump
run
haste

File: legnardotmnt.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+class: leonardotmnt
+costumes:
+- chatcolor: '&1'
+  name: &2Leo&1nar&2do&8(&2TMNT&8)
```
Comment (TheLightShadow) [inline legnardotmnt.yml]: Change the '&' symbols to the § symbols, you can copy from this message and just replace them.

File: legnardotmnt.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  sig: nyJN+e+QNIGydYedtvlnO8FxVE5LJ/mmA7Us2M839Cnt7h4M11QFEfD/6i5QCEGG7mLDjEJr3jGBLLgAaz3BkcwqORUKn2NRuzMlDM8mOavuc9yfaufIL9BNLD0YngZyTwTk5cNH1oQRUkeoGvw/8NmphFoHbKQeWO3EpHjZxClx8jaKNIgPP+qpkydtYbEcCLtnu86yNAN4hImjHJin2hqFa/7vJbd/COwTRx+GjrocKThZvodj3vbfzh+dVyE/oUhPj1NGjdpjHG9C5YdnwB7D4Nz7P8ThtdhtDZ3YnRM3PJ8Bu7p0w+y+CqYQge/GN1phyWW3rGlKX/bEyytgsj2z6lnRRyhRfSu+nJt3hnPONWQntsgQtcaJTrc9wkS+kvO8jCrgw4c+QW5Q467cdMozIFai/sDKnSX7hIeepZ/RN8lRTwRLqzlxUsV9ZjMjXbCp/8zI72WPPk0LmLrPqK3JuLDwvuM0MVDTSnp3zexnCH2ryymi99wQsDGwHwV0mE4IxY6eElfG2ghDcIa95OUYEkFOl6r7KujW1JNBh/ix3pkGohwUeIywaffoGV/gd0WV9sd8/D/jqOFEhTQCkmu3D9v1WDZuo8Rp0TyV0gkKu67ivYEwxTHDkG+Zd0Ug5yr30L1o29q/Efpc7Uhpvv1dZb3npUZeZzOgLJt9X70=
+  val: eyJ0aW1lc3RhbXAiOjE1MjMzMzEzNjIzNTMsInByb2ZpbGVJZCI6ImVmN2JjYzk2M2FjMDRkYzZhYWU3MDE5ZTVjNmRkYTJlIiwicHJvZmlsZU5hbWUiOiJzdXBlcmlvcnhtYW4iLCJzaWduYXR1cmVSZXF1aXJlZCI6dHJ1ZSwidGV4dHVyZXMiOnsiU0tJTiI6eyJ1cmwiOiJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlLzg2NWYyMjE0YWMxMmQzODgzMjU1ODJiMmVmNWMzMDc1OTVhNTVkM2M1ZjhjYTUxODIyMWI0YzdhOGEyMzZjIn19fQ==
+effects:
```
Comment (TheLightShadow) [inline legnardotmnt.yml]: Since there are no effects please change it to this:

effects: [
  ]

File: legnardotmnt.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+items:
+- /give <plr> minecraft:iron_sword 1 16:10 name:&1Leonardo's_&fKatana
+- /give <plr> minecraft:iron_sword 1 16:10 name:&1Leonardo's_&fKatana
+jump: 4
+maxHearts: 20
+permissions:
+  superherocity:
+  - ch.alias.bodyslide
+powers:
+- /effects
```
Comment (TheLightShadow) [inline legnardotmnt.yml]: You can remove /effects from powers since the character has no effects

File: legnardotmnt.yml (hunk 1 above)
Comment (malonnnn) [reply legnardotmnt.yml]: confirmed correct

File: legnardotmnt.yml (hunk 2 above)
Comment (malonnnn) [reply legnardotmnt.yml]: lightshadow is correct

you can also use

effects: []

File: legnardotmnt.yml (hunk 3 above)
Comment (malonnnn) [reply legnardotmnt.yml]: lightshadow is correct

**Comment by superiorxman** [comment]
done

**Pull request merged**